storage/src/persistent/database.rs

```diff
@@ -148,25 +142,6 @@ impl<S: KeyValueSchema> KeyValueStoreWithSchema<S> for DB {
 
         Ok(IteratorWithSchema(self.prefix_iterator_cf(cf, key), PhantomData))
     }
-
-    fn contains(&self, key: &S::Key) -> Result<bool, DBError> {
```

Comment:
@michal-stanek 
I would leave `contains` here, and just implement it with get here, something like:
```
fn contains(&self, key: &S::Key) -> Result<bool, DBError> {
    get(key).is_some()
}
```

maybe in the future, there will be better rocksdb way to do it than just with get,
and also it will be much readable to have distinction where do you need `get` or just `contains`, 
maybe get touches data and contains not

Reply:
ok, I'll wrap it like that